Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -15,13 +17,14 @@ function defineListeners(cy, path, unixName) {
 		var node = e.cyTarget;
 		conceptDescription(path, node);
 		getTexts(node, path, unixName);
+		getEntireText(node,path);
```

Comment:
this would mean that when a user clicks on a node, the function getEntireText should be called. Which it shouldn't. The text should be shown when the user clicks on a link in the side bar, not when clicking on a node.